modules/nw-multus-ipam-object.adoc

```diff
@@ -310,4 +310,78 @@ pod/whereabouts-reconciler-p4sxw 1/1 Running 0 6s
 pod/whereabouts-reconciler-rvfdv 1/1 Running 0 6s
 pod/whereabouts-reconciler-svzw9 1/1 Running 0 6s
 daemonset.apps/whereabouts-reconciler 6 6 6 6 6 kubernetes.io/os=linux 6s
+----
+
+[id="nw-multus-configuring-whereabouts-ip-reconciler-schedule_{context}"]
+== Configuring the Whereabouts IP reconciler schedule
+
+The Whereabouts IPAM CNI plugin runs the IP reconciler daily to clean up any stranded IP allocations that may result in exhausting IPs and therefore prevent new Pods from getting an IP allocated to them.
+
+The following procedure shows how to change the frequency at which the IP reconciler runs. 
```

Comment:
```suggestion
The following procedure shows how to change the frequency at which the IP reconciler runs. 
```
Try to avoid self referential [language](Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".

).